src/views/CreateWorkPackageModal.vue

```diff
@@ -0,0 +1,591 @@
+<template>
+	<NcModal v-if="openModal"
+		class="create-workpackage-modal"
+		:can-close="true"
+		@close="closeModal">
+		<h2 class="create-workpackage-modal--title">
+			{{ t('integration_openproject', 'Create and link a new work package') }}
+		</h2>
+		<div class="create-workpackage-form-wrapper">
+			<div class="create-workpackage-form--label">
+				{{ t('integration_openproject', 'Project *') }}
+			</div>
+			<NcSelect ref="createWorkPackageInput"
+				class="create-workpackage-form--select"
+				input-id="createWorkPackageInput"
+				data-test-id="available-projects"
+				:placeholder="t('integration_openproject', 'Select a project')"
+				:options="mappedNodes"
+				:filterable="true"
+				:close-on-select="true"
+				:clear-search-on-blur="() => false"
+				:append-to-body="false"
+				:value="project"
+				@option:selected="onSelectProject">
+				<template #option="{ label, relation, counter }">
+					<span v-if="relation === 'child'" :style="{paddingLeft: counter + 'em' }" />
+					<span>{{ label }}</span>
+				</template>
+			</NcSelect>
+			<p v-if="error.error && error.attribute === 'project'" class="validation-error">
+				{{ t('integration_openproject', error.message) }}
```

Comment:
this kind of things will not be translated, because transifex does not execute the code and so does not know what the content of `error.message` is

```suggestion
				{{ error.message }}
```

if we want to have translatable error messages, we will have to "rewrite" them like was done in https://github.com/nextcloud/integration_openproject/blob/54d2c2e3603a248ea71e8d931f70abb60473aa80/src/views/Dashboard.vue#L229

Reply:
Didn't know that. Thanks I'll do it like it's done in the Dashboard

Reply:
the error message is directly from openproject and upon closer I think we need it to be dynamic so I have opted out of translating it